src/doc/src/reference/semver.md

```diff
@@ -206,6 +208,111 @@ This is not considered a major change because conventionally glob imports are
 a known forwards-compatibility hazard. Glob imports of items from external
 crates should be avoided.
 
+<a id="type-layout"></a>
+### Major: Changing the alignment, layout, or size of a well-defined type
+
+It is a breaking change to change the alignment, layout, or size of a type that was previously well-defined.
+
+In general, nominal types that use the [the default representation] do not have a well-defined alignment, layout, or size.
```

Comment:
What distinction is "nominal types" meant to make, here? Unnamed types, like tuples, that use the default representation (which is the only option we provide for them as far as I know) *also* don't have a well-defined alignment, layout, or size.

Reply:
I was mostly trying to emphasize that this discussion was around types that accept the `#[repr]` attribute. However, it is not important and I dropped the word.